### PR TITLE
Changes to landice COMPASS tests for new conda env

### PR DESCRIPTION
**Pull request by matthewhoffman**
This merge changes the landice COMPASS tests to make use of the tools that are now available through a compass conda package.  It also updates all python scripts in the COMPASS landice directory to be compatible with python 3.

### CODE REVIEW

File: testing_and_setup/compass/general.config.landice [hunk 1]
```diff
@@ -25,11 +25,9 @@ forward = FULL_PATH_TO_FORWARD_TEMPLATE_STREAMS
...
 [executables]
 model = FULL_PATH_TO_MODEL
-mesh_converter = FULL_PATH_TO_MESH_CONVERTER
-cell_culler = FULL_PATH_TO_CELL_CULLER
-metis = FULL_PATH_TO_METIS
-grid_to_li_grid = FULL_PATH_TO_create_landice_grid_from_generic_MPAS_grid.py
-periodic_hex = FULL_PATH_TO_PERIODIC_HEX
+grid_to_li_grid = FULL_PATH_TO/MPAS-Tools/landice/mesh_tools_li/create_landice_grid_from_generic_MPAS_grid.py
+define_cull_mask = FULL_PATH_TO/MPAS-Tools/landice/mesh_tools_li/define_cullMask.py
+interpolate_to_mpasli_grid = FULL_PATH_TO/MPAS-Tools/landice/mesh_tools_li/interpolate_to_mpasli_grid.py
```
Comment (xylar) [inline testing_and_setup/compass/general.config.landice]: Presumably these 3 scripts should just be in the path when you load the `compass` environment.  That's basically the reason for the reorganization.  Is the plan to remove these 3 lines once the `compass` environment gets updated after https://github.com/MPAS-Dev/MPAS-Tools/pull/260 gets merged?

Comment (matthewhoffman) [reply testing_and_setup/compass/general.config.landice]: Yes, I only started updating the LI COMPASS - there will be a whole raft of additional commits here once I have merged the MPAS-Tools updates.

Comment (xylar) [reply testing_and_setup/compass/general.config.landice]: Perfect! Just wanted to be sure.

**Comment by xylar** [comment]
@matthewhoffman, I just updated the `compass_py3.7` environment on IC to include `mpas_tools=0.0.3`.  To use it, just run:
```
source /usr/projects/climate/SHARED_CLIMATE/anaconda_envs/base/etc/profile.d/conda.sh
conda activate compass_py3.7
```

It might be worth creating an equivalent to:
https://github.com/MPAS-Dev/MPAS-Model/blob/ocean/develop/testing_and_setup/compass/README_ocean.md
for land ice, but that's obviously up to you.

**Pull request merged**